src/config/settings.ts

```diff
@@ -38,6 +38,8 @@ const defaultSettings = {
   updater:        true,
 };
 
+export const defaultTestConfig = typeof defaultSettings;
```

Comment:
So `defaultTestConfig` is a _type_ (with no value)? I was thinking of just exporting `defaultSettings` (line 23) directly; would that work?

VSCode claims that `defaultTestConfig` is one of a bunch of strings (`"string" | "number" | …`), but doesn't tell me the actual value.

Reply:
Done! 
Thanks for the tips.

Reply:
I don't think we still need this export?

Reply:
omg...
I thought that I had removed it, but I was checked out on another branch ...
Arriving on Vancouver, I'll pay the beer :D